packages/pointer_interceptor/pointer_interceptor_web/example/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: pointer_interceptor_web_example
+description: "Demonstrates how to use the pointer_interceptor_web plugin."
+publish_to: 'none'
+
+environment:
+  sdk: ">=3.1.0 <4.0.0"
+  flutter: ">=3.13.0"
+
+dependencies:
+  cupertino_icons: ^1.0.2
+  flutter:
+    sdk: flutter
+  pointer_interceptor:
```

Comment:
And this removed, in favor of a normal pointer_interceptor_platform_interface dependency.